openownership/data-standard
# Valid data directory: remove fermentcat_  and tecido_ filename prefixes

# Overview

- What does this pull request do?

In the tests/data/valid-statements directory, all files with a fermentcat_ or tecido_ prefix have had that prefix removed.

- How can a reviewer test or examine your changes?

Run pytest tests/ to see that all the files are still checked and remain valid.

- Who is best placed to review it?

@radix0000 this week.


## Translations

n/a

## Documentation & Release

n/a
